### PR TITLE
Use express-graphql

### PR DESCRIPTION
**Pull request by ironsidevsquincy**
And remove unused routes

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -1,2 +1,24 @@
...
+const healthChecks = nHealth(path.resolve(__dirname, './config/health-checks'));
+const app = express({
+	healthChecks: healthChecks.asArray(),
+	systemCode: 'next-api'
+});
+
+app.get('/__gtg', gtg);
+
+app.use(authenticate, cacheControl);
+app.use('/v1/query', query);
```
Comment (andygout) [inline server/app.js]: Does this mean we are still going to do versioning despite [GraphQL docs' recommendations](http://graphql.org/learn/best-practices/#versioning)?

**Comment by andygout** [comment]
How does `express-graphql` set the API key which is presumably required to use the playground?

**Comment by ironsidevsquincy** [comment]
Playground uses the s3o-middleware. Currently the whole api does, just so we can get this out. Will need to get the API key stuff working

**Comment by andygout** [comment]
👍  (seem to have lost option to 'review'...)

**Comment by ironsidevsquincy** [comment]
Cool. Once the config vars is all set up (and therefore building in CI), I can merge

**Pull request merged**